v2/emailpassword/quick-setup/frontend.mdx

```diff
@@ -51,12 +52,12 @@ import Session from "supertokens-auth-react/recipe/session";
 SuperTokens.init({
     appInfo: {
         // learn more about this on https://supertokens.io/docs/emailpassword/appinfo
-        appName: "YOUR APP NAME",
-        apiDomain: "YOUR API DOMAIN",
-        websiteDomain: "YOUR WEBSITE DOMAIN",
-        // The default values for apiBasePath and websiteBasePath are /auth
-        apiBasePath: "YOUR API BASE PATH",
-        websiteBasePath: "YOUR WEBSITE BASE PATH"
+        // Compulsory
+        appName: "YOUR APP NAME", // Example: "SuperTokens",
+        apiDomain: "YOUR API DOMAIN", // Example: "https://api.supertokens.io",
+        websiteDomain: "YOUR WEBSITE DOMAIN", // Example: "https://supertokens.io"
+        // Optional
+        apiBasePath: "YOUR API BASE PATH", // Example: "/auth"
```

Comment:
Remove optional and apiBasePath from frontend / backend eveyrhwere.. also the Compulsory thing. Essentially, there shuold be no diff for this section (except for the important tool tip).